docs/accessing_the_cluster.md

```diff
@@ -0,0 +1,79 @@
+# Accessing the Cluster
+
+## Using the Kubernetes proxy to access the cluster
+Information about the cluster can be accessed by using a proxy URL and by providing the keys to the cluster.
+For example, for a cluster that has cluster level logging enabled using Elasticsearch you can fetch information about
+the Elasticsearch logging cluster.
+
+First, you will need to obtain the keys (username and password) for your cluster:
+
+```
+$ cat ~/.kube/kubernetes-satnam2_kubernetes/kubernetes_auth
+{
+  "User": "admin",
+  "Password": "4mty0Vl9nNFfwLJz",
+  "CAFile": "/Users/satnam/.kube/kubernetes-satnam2_kubernetes/kubernetes.ca.crt",
+  "CertFile": "/Users/satnam/.kube/kubernetes-satnam2_kubernetes/kubecfg.crt",
+  "KeyFile": "/Users/satnam/.kube/kubernetes-satnam2_kubernetes/kubecfg.key"
+}
+```
+
+To access a service endpoint `/A/B/C` via the proxy service for your service S you need to specify an HTTPS address
+for the Kubernetes master followed by `/api/v1beta1/proxy/S/A/B/C/`. Currently it is important to
```

Comment:
Please include the namespace in the path, so users can tell where it should go. Using the default namespace is fine.